### PR TITLE
fixed scrolling in ie11

### LINKED ISSUE #4167: showColumns cannot scroll down the sidebar in IE11
By creating a bootstrap table with `showColumns: true` table option, user cannot scroll down the sidebar in Internet explorer 11 browser.

Try to scroll by dragging the bar, it won't work. Only by clicking the arrows or using mouse wheel.

### PR DESCRIPTION
**Pull request by UtechtDustin**
Fix #4167

### CODE REVIEW

**Comment by wenzhixin** [comment]
I think it's not a good way to solve the problem because it will always display the scrollbar (for example in Chrome):
image

Maybe we can solve it with this answer:
https://stackoverflow.com/questions/45890061/overflow-y-scroll-not-working-in-ie-11

**Comment by UtechtDustin** [comment]
If the scrollbar is visible in chrome we have a overflow issue or not ?
I will check it later!

**Comment by UtechtDustin** [comment]
reverted the commit and used the bugfix from stackoverflow.
Tested in ie11, firefox and chrome.

File: src/themes/theme.scss [hunk 1]
```diff
@@ -38,6 +38,7 @@
         text-align: left;
         max-height: 300px;
         overflow: auto;
+        -ms-overflow-style:scrollbar;
```
Comment (wenzhixin) [inline src/themes/theme.scss]: Add an space ` ` after `:`
```
-ms-overflow-style: scrollbar;
```

Comment (UtechtDustin) [reply src/themes/theme.scss]: done

**Pull request merged**